api/helpers/unlock-game.js

```diff
@@ -14,6 +14,10 @@ module.exports = {
   },
 
   fn: async ({ lock }, exits) => {
+    if (!lock) {
+      return exits.success();
+    }
```

Comment:
No-opping when an empty lock is provided lets us skip this check in callers of lock-game